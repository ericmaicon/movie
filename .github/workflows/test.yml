name: Lint & Test

on: push

jobs:
  test:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: '14'
      - name: Cache dependencies
        uses: actions/cache@v2.1.4
        id: cache-dependencies
        with:
          path: |
            node_modules
            */*/node_modules
          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
      - name: Install Dependencies
        if: steps.cache-dependencies.outputs.cache-hit != 'true'
        run: yarn install && yarn bootstrap
      - name: Link
        run: yarn lerna link
      - name: Build deps
        run: yarn build
      - name: Lint
        run: yarn lint
      - name: Run Tests
        run: yarn test
      - uses: codecov/codecov-action@v1
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
          directory: packages/**/coverage
          fail_ci_if_error: true
