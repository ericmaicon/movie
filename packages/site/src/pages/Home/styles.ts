import { Col } from 'react-grid-system';
import { AiOutlineFire } from 'react-icons/ai';

import styled from 'styled-components';

export const HeaderContainer = styled(Col)`
  display: flex;
  flex-direction: row;
  margin-bottom: 20px;
  margin-top: 50px;
`;

export const StyledFire = styled(AiOutlineFire)`
  color: ${({ theme }) => theme.colors.red};
  font-size: 1rem;

  //tablet
  @media (min-width: 768px) and (max-width: 1024px) {
    font-size: 1.5rem;
  }

  //desktop
  @media (min-width: 1025px) {
    font-size: 2rem;
  }
`;
