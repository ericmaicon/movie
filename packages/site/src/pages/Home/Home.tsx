import React from 'react';
import { Container, Row, Col } from 'react-grid-system';
import Skeleton from 'react-loading-skeleton';

import { Movie } from '@movie/core';
import { Heading, VideoGallery } from '@movie/uikit';

import * as Styled from './styles';

interface HomeProps {
  onSelectItem: (id: number | string) => Promise<void>;

  items: Movie[] | null;
}

export function Home({ onSelectItem, items, ...rest }: HomeProps) {
  return (
    <div {...rest}>
      <Container>
        <Row>
          <Styled.HeaderContainer sm={12}>
            <Styled.StyledFire /> <Heading size={4}>Popular Movies</Heading>
          </Styled.HeaderContainer>
        </Row>
        <Row>
          <Col sm={12}>
            {items ? (
              <VideoGallery items={items} onSelectItem={onSelectItem} />
            ) : (
              <Skeleton height={150} />
            )}
          </Col>
        </Row>
      </Container>
    </div>
  );
}

export default Home;
