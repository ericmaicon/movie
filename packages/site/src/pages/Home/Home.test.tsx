import React from 'react';

import userEvent from '@testing-library/user-event';
import snapshotDiff from 'snapshot-diff';

import { fireEvent, render, waitFor } from '~/test/index';

import { Home } from './Home';

const items = [
  {
    id: 1,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/drulhSX7P5TQlEMQZ3JoXKSDEfz.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
  {
    id: 2,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/srYya1ZlI97Au4jUYAktDe3avyA.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
  {
    id: 3,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/6TPZSJ06OEXeelx1U1VIAt0j9Ry.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
  {
    id: 4,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/dWSnsAGTfc8U27bWsy2RfwZs0Bs.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
  {
    id: 5,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/vfuzELmhBjBTswXj2Vqxnu5ge4g.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
  {
    id: 6,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/c7VlGCCgM9GZivKSzBgzuOVxQn7.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
  {
    id: 7,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/lOSdUkGQmbAl5JQ3QoHqBZUbZhC.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
  {
    id: 8,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/6XYLiMxHAaCsoyrVo38LBWMw2p8.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
  {
    id: 9,
    image:
      'https://www.themoviedb.org/t/p/w220_and_h330_face/h9DIlghaiTxbQbt1FIwKNbQvEL.jpg',
    title:
      'When the crew of a space junk collector ship called The Victory discovers a humanoid..',
    link: '',
  },
];

describe('Home', () => {
  it('should render correctly', () => {
    const { getByTestId } = render(
      <Home onFilter={jest.fn()} data-testid="home" items={items} />,
    );

    const homeElement = getByTestId('home');

    expect(homeElement).toBeInTheDocument();
    expect(homeElement).toMatchSnapshot();
  });

  it('should render loading when there is no items', () => {
    const { rerender, asFragment, getByTestId } = render(
      <Home onFilter={jest.fn()} data-testid="home" />,
    );

    const loading = asFragment();
    rerender(<Home onFilter={jest.fn()} data-testid="home" items={items} />);
    const withItems = asFragment();

    expect(getByTestId('home')).toBeDisabled();
    expect(withItems).toMatchSnapshot();
    expect(snapshotDiff(loading, withItems)).toMatchSnapshot();
  });

  it('should call onFilter after submit form', async () => {
    const onFilter = jest.fn();
    const { getByRole, getByTestId } = render(
      <Home onFilter={onFilter} items={items} />,
    );

    const searchInput = getByRole('textbox');
    const formElement = getByTestId('form');

    userEvent.type(searchInput, 'Search');
    fireEvent.submit(formElement);
    await waitFor(() =>
      expect(onFilter).toHaveBeenCalledWith({
        search: 'Search',
      }),
    );
  });
});
