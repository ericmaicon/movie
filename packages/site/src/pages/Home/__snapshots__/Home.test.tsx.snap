// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
.c2 {
  margin: 0px;
  font-size: 1rem;
  font-family: Roboto;
  font-weight: 700;
  font-size: 4rem;
  line-height: 1em;
  -webkit-letter-spacing: 0em;
  -moz-letter-spacing: 0em;
  -ms-letter-spacing: 0em;
  letter-spacing: 0em;
}

.c4 {
  cursor: pointer;
  width: 100%;
}

.c4:hover {
  box-shadow: 0px 0px 0px 2px rgba(0,0,0,0.2);
}

.c3 {
  display: grid;
  grid-template-columns: repeat(6,1fr);
  grid-gap: 15px;
}

.c3 img:first-child {
  grid-column-start: 1;
  grid-column-end: 3;
  grid-row-start: 1;
  grid-row-end: 3;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  margin-bottom: 20px;
  margin-top: 50px;
}

.c1 {
  color: #A62218;
  font-size: 1rem;
}

@media (min-width:768px) and (max-width:1024px) {
  .c2 {
    font-size: 1.5rem;
  }
}

@media (min-width:1025px) {
  .c2 {
    font-size: 2rem;
  }
}

@media (min-width:768px) and (max-width:1024px) {
  .c1 {
    font-size: 1.5rem;
  }
}

@media (min-width:1025px) {
  .c1 {
    font-size: 2rem;
  }
}

<div
  data-testid="home"
>
  <div
    style="box-sizing: border-box; position: relative; margin-left: auto; margin-right: auto; padding-left: 15px; padding-right: 15px; max-width: 960px;"
  >
    <div
      style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
    >
      <div
        class="c0"
        style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 100%; flex-basis: 100%; flex-grow: 0; flex-shrink: 0; max-width: 100%; margin-left: 0%;"
      >
        <svg
          class="c1"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M834.1 469.2A347.49 347.49 0 0 0 751.2 354l-29.1-26.7a8.09 8.09 0 0 0-13 3.3l-13 37.3c-8.1 23.4-23 47.3-44.1 70.8-1.4 1.5-3 1.9-4.1 2-1.1.1-2.8-.1-4.3-1.5-1.4-1.2-2.1-3-2-4.8 3.7-60.2-14.3-128.1-53.7-202C555.3 171 510 123.1 453.4 89.7l-41.3-24.3c-5.4-3.2-12.3 1-12 7.3l2.2 48c1.5 32.8-2.3 61.8-11.3 85.9-11 29.5-26.8 56.9-47 81.5a295.64 295.64 0 0 1-47.5 46.1 352.6 352.6 0 0 0-100.3 121.5A347.75 347.75 0 0 0 160 610c0 47.2 9.3 92.9 27.7 136a349.4 349.4 0 0 0 75.5 110.9c32.4 32 70 57.2 111.9 74.7C418.5 949.8 464.5 959 512 959s93.5-9.2 136.9-27.3A348.6 348.6 0 0 0 760.8 857c32.4-32 57.8-69.4 75.5-110.9a344.2 344.2 0 0 0 27.7-136c0-48.8-10-96.2-29.9-140.9zM713 808.5c-53.7 53.2-125 82.4-201 82.4s-147.3-29.2-201-82.4c-53.5-53.1-83-123.5-83-198.4 0-43.5 9.8-85.2 29.1-124 18.8-37.9 46.8-71.8 80.8-97.9a349.6 349.6 0 0 0 58.6-56.8c25-30.5 44.6-64.5 58.2-101a240 240 0 0 0 12.1-46.5c24.1 22.2 44.3 49 61.2 80.4 33.4 62.6 48.8 118.3 45.8 165.7a74.01 74.01 0 0 0 24.4 59.8 73.36 73.36 0 0 0 53.4 18.8c19.7-1 37.8-9.7 51-24.4 13.3-14.9 24.8-30.1 34.4-45.6 14 17.9 25.7 37.4 35 58.4 15.9 35.8 24 73.9 24 113.1 0 74.9-29.5 145.4-83 198.4z"
          />
        </svg>
         
        <h4
          class="c2"
          size="4"
        >
          Popular Movies
        </h4>
      </div>
    </div>
    <div
      style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
    >
      <div
        style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 100%; flex-basis: 100%; flex-grow: 0; flex-shrink: 0; max-width: 100%; margin-left: 0%;"
      >
        <div
          class="c3"
        >
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/drulhSX7P5TQlEMQZ3JoXKSDEfz.jpg"
          />
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/srYya1ZlI97Au4jUYAktDe3avyA.jpg"
          />
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/6TPZSJ06OEXeelx1U1VIAt0j9Ry.jpg"
          />
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/dWSnsAGTfc8U27bWsy2RfwZs0Bs.jpg"
          />
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/vfuzELmhBjBTswXj2Vqxnu5ge4g.jpg"
          />
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/c7VlGCCgM9GZivKSzBgzuOVxQn7.jpg"
          />
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/lOSdUkGQmbAl5JQ3QoHqBZUbZhC.jpg"
          />
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/6XYLiMxHAaCsoyrVo38LBWMw2p8.jpg"
          />
          <img
            alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
            class="c4"
            data-testid="video-card"
            src="https://www.themoviedb.org/t/p/w220_and_h330_face/h9DIlghaiTxbQbt1FIwKNbQvEL.jpg"
          />
        </div>
      </div>
    </div>
    <span
      style="display: table; clear: both;"
    />
  </div>
</div>
`;

exports[`Home should render loading when there is no items 1`] = `
<DocumentFragment>
  .c2 {
  margin: 0px;
  font-size: 1rem;
  font-family: Roboto;
  font-weight: 700;
  font-size: 4rem;
  line-height: 1em;
  -webkit-letter-spacing: 0em;
  -moz-letter-spacing: 0em;
  -ms-letter-spacing: 0em;
  letter-spacing: 0em;
}

.c4 {
  cursor: pointer;
  width: 100%;
}

.c4:hover {
  box-shadow: 0px 0px 0px 2px rgba(0,0,0,0.2);
}

.c3 {
  display: grid;
  grid-template-columns: repeat(6,1fr);
  grid-gap: 15px;
}

.c3 img:first-child {
  grid-column-start: 1;
  grid-column-end: 3;
  grid-row-start: 1;
  grid-row-end: 3;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  margin-bottom: 20px;
  margin-top: 50px;
}

.c1 {
  color: #A62218;
  font-size: 1rem;
}

@media (min-width:768px) and (max-width:1024px) {
  .c2 {
    font-size: 1.5rem;
  }
}

@media (min-width:1025px) {
  .c2 {
    font-size: 2rem;
  }
}

@media (min-width:768px) and (max-width:1024px) {
  .c1 {
    font-size: 1.5rem;
  }
}

@media (min-width:1025px) {
  .c1 {
    font-size: 2rem;
  }
}

<div
    data-testid="home"
  >
    <div
      style="box-sizing: border-box; position: relative; margin-left: auto; margin-right: auto; padding-left: 15px; padding-right: 15px; max-width: 960px;"
    >
      <div
        style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
      >
        <div
          class="c0"
          style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 100%; flex-basis: 100%; flex-grow: 0; flex-shrink: 0; max-width: 100%; margin-left: 0%;"
        >
          <svg
            class="c1"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 1024 1024"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M834.1 469.2A347.49 347.49 0 0 0 751.2 354l-29.1-26.7a8.09 8.09 0 0 0-13 3.3l-13 37.3c-8.1 23.4-23 47.3-44.1 70.8-1.4 1.5-3 1.9-4.1 2-1.1.1-2.8-.1-4.3-1.5-1.4-1.2-2.1-3-2-4.8 3.7-60.2-14.3-128.1-53.7-202C555.3 171 510 123.1 453.4 89.7l-41.3-24.3c-5.4-3.2-12.3 1-12 7.3l2.2 48c1.5 32.8-2.3 61.8-11.3 85.9-11 29.5-26.8 56.9-47 81.5a295.64 295.64 0 0 1-47.5 46.1 352.6 352.6 0 0 0-100.3 121.5A347.75 347.75 0 0 0 160 610c0 47.2 9.3 92.9 27.7 136a349.4 349.4 0 0 0 75.5 110.9c32.4 32 70 57.2 111.9 74.7C418.5 949.8 464.5 959 512 959s93.5-9.2 136.9-27.3A348.6 348.6 0 0 0 760.8 857c32.4-32 57.8-69.4 75.5-110.9a344.2 344.2 0 0 0 27.7-136c0-48.8-10-96.2-29.9-140.9zM713 808.5c-53.7 53.2-125 82.4-201 82.4s-147.3-29.2-201-82.4c-53.5-53.1-83-123.5-83-198.4 0-43.5 9.8-85.2 29.1-124 18.8-37.9 46.8-71.8 80.8-97.9a349.6 349.6 0 0 0 58.6-56.8c25-30.5 44.6-64.5 58.2-101a240 240 0 0 0 12.1-46.5c24.1 22.2 44.3 49 61.2 80.4 33.4 62.6 48.8 118.3 45.8 165.7a74.01 74.01 0 0 0 24.4 59.8 73.36 73.36 0 0 0 53.4 18.8c19.7-1 37.8-9.7 51-24.4 13.3-14.9 24.8-30.1 34.4-45.6 14 17.9 25.7 37.4 35 58.4 15.9 35.8 24 73.9 24 113.1 0 74.9-29.5 145.4-83 198.4z"
            />
          </svg>
           
          <h4
            class="c2"
            size="4"
          >
            Popular Movies
          </h4>
        </div>
      </div>
      <div
        style="margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;"
      >
        <div
          style="box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 100%; flex-basis: 100%; flex-grow: 0; flex-shrink: 0; max-width: 100%; margin-left: 0%;"
        >
          <div
            class="c3"
          >
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/drulhSX7P5TQlEMQZ3JoXKSDEfz.jpg"
            />
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/srYya1ZlI97Au4jUYAktDe3avyA.jpg"
            />
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/6TPZSJ06OEXeelx1U1VIAt0j9Ry.jpg"
            />
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/dWSnsAGTfc8U27bWsy2RfwZs0Bs.jpg"
            />
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/vfuzELmhBjBTswXj2Vqxnu5ge4g.jpg"
            />
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/c7VlGCCgM9GZivKSzBgzuOVxQn7.jpg"
            />
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/lOSdUkGQmbAl5JQ3QoHqBZUbZhC.jpg"
            />
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/6XYLiMxHAaCsoyrVo38LBWMw2p8.jpg"
            />
            <img
              alt="When the crew of a space junk collector ship called The Victory discovers a humanoid.."
              class="c4"
              data-testid="video-card"
              src="https://www.themoviedb.org/t/p/w220_and_h330_face/h9DIlghaiTxbQbt1FIwKNbQvEL.jpg"
            />
          </div>
        </div>
      </div>
      <span
        style="display: table; clear: both;"
      />
    </div>
  </div>
</DocumentFragment>
`;

exports[`Home should render loading when there is no items 2`] = `
"Snapshot Diff:
- First value
+ Second value

@@ -39,18 +39,68 @@
          style=\\"margin-left: -15px; margin-right: -15px; display: flex; flex-wrap: wrap; flex-grow: 0; flex-shrink: 0; align-items: normal; justify-content: flex-start;\\"
        >
          <div
            style=\\"box-sizing: border-box; min-height: 1px; position: relative; padding-left: 15px; padding-right: 15px; width: 100%; flex-basis: 100%; flex-grow: 0; flex-shrink: 0; max-width: 100%; margin-left: 0%;\\"
          >
-           <span>
-             <span
-               class=\\"react-loading-skeleton css-1vmnjpn-skeletonStyles-Skeleton\\"
-               style=\\"height: 150px;\\"
+           <div
+             class=\\"sc-dQppl ebSKzZ\\"
            >
-               ‌
-             </span>
-           </span>
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/drulhSX7P5TQlEMQZ3JoXKSDEfz.jpg\\"
+             />
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/srYya1ZlI97Au4jUYAktDe3avyA.jpg\\"
+             />
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/6TPZSJ06OEXeelx1U1VIAt0j9Ry.jpg\\"
+             />
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/dWSnsAGTfc8U27bWsy2RfwZs0Bs.jpg\\"
+             />
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/vfuzELmhBjBTswXj2Vqxnu5ge4g.jpg\\"
+             />
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/c7VlGCCgM9GZivKSzBgzuOVxQn7.jpg\\"
+             />
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/lOSdUkGQmbAl5JQ3QoHqBZUbZhC.jpg\\"
+             />
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/6XYLiMxHAaCsoyrVo38LBWMw2p8.jpg\\"
+             />
+             <img
+               alt=\\"When the crew of a space junk collector ship called The Victory discovers a humanoid..\\"
+               class=\\"sc-crrsfI bYspdc\\"
+               data-testid=\\"video-card\\"
+               src=\\"https://www.themoviedb.org/t/p/w220_and_h330_face/h9DIlghaiTxbQbt1FIwKNbQvEL.jpg\\"
+             />
+           </div>
          </div>
        </div>
        <span
          style=\\"display: table; clear: both;\\"
        />"
`;
